### PR TITLE
docs(cards): Adding styling section to cards topic

### PR DESCRIPTION
**Pull request by astaev**
https://github.com/IgniteUI/igniteui-docfx/issues/1275

### CODE REVIEW

File: en/components/card.md [hunk 1]
```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
...
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$yellow-color: #e7eb0b;
+$black-color: #f00b1e;
+
+$dark-palette: igx-palette($primary: $black-color, $secondary: $yellow-color);
+```
+
+And then with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) we can easily retrieve color from the palette.
```
Comment (tachojelev) [inline en/components/card.md]: The snippet below should use the **igx-color** function.

File: en/components/card.md [hunk 2]
```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
...
+    $palette: $dark-palette,
+    $background:#fd6bd93a,
+    $header-text-color:#f2f607,
+    $subtitle-text-color:#fa2509,
+    $content-text-color:#0a41f7
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please refer to [`Palettes`](../themes/palette.md) topic for detailed guidance on how to use them.
```
Comment (tachojelev) [inline en/components/card.md]: The **Palettes** hyperlink does not work.

File: en/components/card.md [hunk 3]
```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
...
+
+#### Using Schemas
+
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/schemas.md). A **schema** is a recipe of a theme.
```
Comment (tachojelev) [inline en/components/card.md]: The **schemas** hyperlink does not work.

File: en/components/card.md [hunk 4]
```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
...
+
+<div class="sample-container loading" style="height: 486px">
+    <iframe id="card-style-iframe" seamless="" width="100%" height="100%" frameborder="0" data-src="{environment:demosBaseUrl}/layouts/card-styling-sample" class="lazyload no-theming">
+</iframe></div>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="card-style-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Summary
+In this article we covered a lot of ground with the card component. First, we created a very simple card with text content only. Then added some images to make the card a bit more appealing. We used some additional Ignite UI for Angular components inside our card, like avatars, buttons and icons, to enrich the experience and add some functionality. And finally, we changed the card's theme by setting some exposed theme colors, creating custom palletes and extending schemas.
```
Comment (tachojelev) [inline en/components/card.md]: Typo: **palletes** should be **palettes**

File: en/components/card.md [hunk 5]
```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
 </igx-card-actions>
 ```
 
-#### Summary
-In this article we covered a lot of ground with the card component. First, we created a very simple card with text content only. Then added some images to make the card a bit more appealing. Finally, we used some additional Ignite UI for Angular components inside our card, like avatars, buttons and icons, to enrich the experience and add some functionality. The card component is capable of displaying more different layouts worth exploring in the Card Demo in the beginning of this article.
+### Styling
```
Comment (tachojelev) [inline en/components/card.md]: We need to update the **jp** version of the file as well.

File: en/components/card.md (hunk 1 above)
Comment (astaev) [reply en/components/card.md]: Fixed

File: en/components/card.md (hunk 4 above)
Comment (astaev) [reply en/components/card.md]: Fixed

File: en/components/card.md (hunk 2 above)
Comment (astaev) [reply en/components/card.md]: Fixed

File: en/components/card.md (hunk 3 above)
Comment (astaev) [reply en/components/card.md]: Fixed

File: en/components/card.md (hunk 5 above)
Comment (astaev) [reply en/components/card.md]: JP file updated.

**Pull request merged**